assets/js/components/KeyMetrics/ChipTabGroup/index.stories.js

```diff
@@ -115,6 +117,38 @@ Default.scenario = {
 	label: 'Components/KeyMetrics/ChipTabGroup/default',
 };
 
+export const WithNewlyDetectedMetrics = Template.bind( {} );
+WithNewlyDetectedMetrics.storyName = 'With Newly Detected Events';
+WithNewlyDetectedMetrics.args = {
+	setupRegistry: ( registry ) => {
+		const selectedMetrics = [
+			KM_ANALYTICS_VISITS_PER_VISITOR,
+			KM_ANALYTICS_VISIT_LENGTH,
+			KM_ANALYTICS_TOP_CITIES_DRIVING_LEADS,
+			KM_ANALYTICS_TOP_TRAFFIC_SOURCE_DRIVING_LEADS,
+		];
+
+		registry.dispatch( CORE_FORMS ).setValues( KEY_METRICS_SELECTION_FORM, {
+			[ KEY_METRICS_SELECTED ]: selectedMetrics,
+			[ EFFECTIVE_SELECTION ]: selectedMetrics,
+		} );
+
+		registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.setDetectedEvents( [ 'contact' ] );
+		registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.receiveConversionReportingInlineData( {
+				newEvents: [ 'contact' ],
+				lostEvents: [],
+			} );
+	},
+	features: [ 'conversionReporting' ],
+};
+WithNewlyDetectedMetrics.scenario = {
+	label: 'Components/KeyMetrics/ChipTabGroup/WithNewlyDetectedMetrics',
+};
```

Comment:
The only thing this story does is add the "New" badge to some metrics. Rather than add this as a new VRT, why not replace the existing VRT? It would cover both with & without the new badge in one VRT.

```suggestion
```

Reply:
Makes sense, will exclude the story and VRT for this scenario, and bled with what is already there